src/Raven.Studio/typescript/viewmodels/resources/createDatabase.ts

```diff
@@ -95,6 +95,12 @@ class createDatabase extends dialogViewModelBase {
 
     private onTopologyLoaded(topology: clusterTopology) {
         this.clusterNodes = topology.nodes();
+        
+        if (this.clusterNodes.length === 1 && this.clusterNodes[0].tag() === "?") {
+            // passive cluster - let's assume node tag is A
+            this.clusterNodes[0].tag("A");
+        }
```

Comment:
In my opinion this is misleading. As if we are in a cluster (especially if we were kick form a cluster)

Reply:
ok I wasn't aware about this case. So what's your recommendation? We get since member with node tag = ?, right after executing raven for a first time. When we select manual node selection and send node tag  = ?, it throws. Maybe we should send '?' anyway but it should be handed serwer side?

Reply:
Yes, passive should be `?`

Reply:
ok, so I reopened issue and this PR goes to trash